services/web/server/src/simcore_service_webserver/folders/_folders_api.py

```diff
@@ -1,341 +1,351 @@
-# pylint: disable=unused-argument
```

Comment:
In this file I changed just few lines, it’s some GitHub glitch. Just open it with a dot, and it works nicely.